### PR TITLE
[Docs] more text for cli

### PR DESCRIPTION
**Pull request by johannesschobel**

### CODE REVIEW

File: docs/prisma2-cli.md [hunk 1]
```diff
@@ -36,30 +36,30 @@ The following environment variables can be provided:
 
 #### `prisma2 init`
 
-Sets up Prisma (i.e. Photon and/or Lift) via an interactive wizard.
+Sets up Prisma (i.e., Photon and/or Lift) via an interactive wizard. You can specify your database connection and/or create a new database to work with. For a list with currently supported database engines, see the [Lift documentation](./lift/use-only-lift.md).
```
Comment (nikolasburk) [inline docs/prisma2-cli.md]: >  For a list with currently supported database engines, see the [Lift documentation](./lift/use-only-lift.md).

Instead of linking to the Lift docs here, I'd suggest to link to this page: https://github.com/prisma/prisma2/blob/master/docs/supported-databases.md (also I'd just call it "databases" instead of "database engines")

File: docs/prisma2-cli.md [hunk 2]
```diff
@@ -36,30 +36,30 @@ The following environment variables can be provided:
...
 
 #### `prisma2 dev`
 
-Starts Prisma [development mode](./development-mode.md).
+Starts Prisma [development mode](./development-mode.md). This opens a web application to interact with your database and defined models. Note that `Prisma Studio` relies on your `prisma.schema` file (and hence the automatically generated Photon client) to provide a fully featured user-interface with CRUD (create / read / update / delete) functionality.
```
Comment (nikolasburk) [inline docs/prisma2-cli.md]: > This opens a web application to interact with your database and defined models.

Not _quite_ true. It starts the web server for Prisma Studio and prints the URL but doesn't _open_ the web app itself. 

> `Prisma Studio`

I wouldn't use inline code formatting (backticks) for "Prisma Studio" here

> `prisma.schema` file

Let's spell this out as "Prisma schema file" (since it can theoretically be named differently). This is also the official notion/terminology we're using throughout the docs.

> (and hence the automatically generated Photon client) 

I'd omit this since that's rather confusing. Yes, Studio uses Photon under the hood but you don't need to have a Photon generator specified in the Prisma schema to use Studio! (at least this shouldn't be the case, if it is it's a bug 🙈)

File: docs/prisma2-cli.md [hunk 3]
```diff
@@ -36,30 +36,30 @@ The following environment variables can be provided:
...
 
 #### `prisma2 generate`
 
-Invokes the generators specified in the Prisma schema file.
+Invokes all generators defined in the `prisma.schema` file. For example, this creates the Photon client to interact with the underlying database. Read more about Photon and its capabilities [here](./photon/use-only-photon.md).
 
 #### `prisma2 introspect`
 
-Introspects the database and generates a data model from it.
+Introspects the database and generates a data model from it. Basically, it analyzes your (already existing) database and automatically creates the `prisma.schema` file for you. This is useful, if you already have an existing application and want to start using Lift.
```
Comment (nikolasburk) [inline docs/prisma2-cli.md]: >  `prisma.schema` file

Same point as before, let's spell it out as "Prisma schema file".

> and want to start using Lift.

this is not only for Lift but also for Photon, so I suggest to rephrase this to: "and want to start using the Prisma Framework"

Comment (nikolasburk) [reply docs/prisma2-cli.md]: also, there's a bit more nuance to this command. the use case you're describing actually is for `prisma2 init`. the `introspect` command is typically used in Photon-only projects where migrations aren't performed through Lift but the Prisma schema needs to be updated. more info here: https://github.com/prisma/prisma2/blob/master/docs/introspection.md

**Review by nikolasburk — CHANGES_REQUESTED** [review]
Thanks a lot for this @johannesschobel 🙏 there are few things that should be updated before we merged this. If possible, it would also be awesome to create a synopsis of the arguments of each command (if it has any), including a description per argument and example invocations for what the command looks like with some arguments! 🙂

File: docs/prisma2-cli.md (hunk 1 above)
Comment (johannesschobel) [reply docs/prisma2-cli.md]: will be fixed in next commit

File: docs/prisma2-cli.md (hunk 2 above)
Comment (johannesschobel) [reply docs/prisma2-cli.md]: will be fixed in next commit

File: docs/prisma2-cli.md (hunk 3 above)
Comment (johannesschobel) [reply docs/prisma2-cli.md]: (hopefully) changed with the next commit

**Comment by johannesschobel** [comment]
Dear @nikolasburk ,
i have just made the suggested changes. Please note that i have **not** added the parameters for the commands.. will work on this tomorrow or so.. 

Feedback is appreciated..

**Comment by nikolasburk** [comment]
Perfect, thanks a ton! 🙏
